src/qt/psbtoperationsdialog.cpp

```diff
@@ -46,20 +46,23 @@ PSBTOperationsDialog::~PSBTOperationsDialog()
 void PSBTOperationsDialog::openWithPSBT(PartiallySignedTransaction psbtx)
 {
     m_transaction_data = psbtx;
-
-    bool complete;
-    size_t n_could_sign;
-    FinalizePSBT(psbtx);  // Make sure all existing signatures are fully combined before checking for completeness.
-    TransactionError err = m_wallet_model->wallet().fillPSBT(SIGHASH_ALL, false /* sign */, true /* bip32derivs */, &n_could_sign, m_transaction_data, complete);
-    if (err != TransactionError::OK) {
-        showStatus(tr("Failed to load transaction: %1")
-            .arg(QString::fromStdString(TransactionErrorString(err).translated)), StatusLevel::ERR);
-        return;
+    if (m_wallet_model) {
+        bool complete;
+        size_t n_could_sign;
+        FinalizePSBT(psbtx); // Make sure all existing signatures are fully combined before checking for completeness.
```

Comment:
`FinalizePSBT` is not a wallet function, it should always be done. It also returns a bool indicating whether the transaction is completely signed, so this could be

    bool complete = FinalizePSBT(psbt);